src/editors/containers/ProblemEditor/components/EditProblemView/AnswerWidget/hooks.js

```diff
@@ -9,7 +9,25 @@ export const state = StrictDict({
   isFeedbackVisible: (val) => useState(val),
 });
 
-export const removeAnswer = ({ answer, dispatch }) => () => {
+export const removeAnswer = ({
+  answers,
+  answer,
+  problemType,
+  dispatch,
+}) => () => {
+  const richTextProblems = [ProblemTypeKeys.SINGLESELECT, ProblemTypeKeys.MULTISELECT];
```

Comment:
It's probably good to define this constant somewhere central, so we don't break a bunch of things when we change which problem types use expanding text areas .

Reply:
That's a good idea